extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -105,7 +105,10 @@ export class BitrateManager {
     onNontrivialWait(video, () => {
       const current = currentSource(video);
       const newBitrate = current.bitrate_ - 1;
-      if (newBitrate >= this.acceptableBitrate_) {
+      if (
+        newBitrate >= this.acceptableBitrate_ ||
+        getBufferedPercentage(video) > BUFFERED_THRESHOLD_PERCENTAGE
```

Comment:
1. Can you add a comment to say that this is to detect a video that's looping?
2. I think we should make this 99% or whatever, I understand that you want to re-use the variable but if a long video buffers towards the end, maybe it's still worth downgrading as it'll positively impact the subsequent videos